platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -239,6 +240,28 @@ whose names differ from the style specification are listed below:
 <% for (const type in renamedProperties) { -%>
 <% if (renamedProperties.hasOwnProperty(type)) { -%>
 
+### <%- camelize(type) %> style functions
+
+The runtime styling API introduced `MGLStyleFunction` to the <%- os %> SDK.
```

Comment:
This reads sort of like a release note. Let's cast this sentence in the present tense and talk about how the API includes style functions rather than introducing them.

Additionally, we have a tendency to use MGLStyleValue as an umbrella for both constants and functions. So even if we talk about the various style function subclasses, I think we should discuss the MGLStyleValue factory method that produces each kind of style function.